lib/stud/task.rb

```diff
@@ -36,5 +36,14 @@ def stop!
     def self.interrupted?
       Thread.current[:stud_task_interrupted]
```

Comment:
guys, the interruption handling stuff in `Stud.interval` and in `Stud#Task` does not make any sense. I will try to make sense of it work on a proper fix. it's useless to just throw random code at this now. see #25 and #26